package-lock.json

```diff
@@ -4051,9 +4063,9 @@
       }
     },
     "@types/react-transition-group": {
-      "version": "4.4.1",
-      "resolved": "https://registry.npmjs.org/@types/react-transition-group/-/react-transition-group-4.4.1.tgz",
-      "integrity": "sha512-vIo69qKKcYoJ8wKCJjwSgCTM+z3chw3g18dkrDfVX665tMH7tmbDxEAnPdey4gTlwZz5QuHGzd+hul0OVZDqqQ==",
+      "version": "4.4.2",
```

Comment:
Can you please revert the changes to the package lock that are not nested under "@edx/paragon" ?  In general, these automated packages could affect something else that isn't part of your change here.  Also, are you sure you need the paragon upgrade to submit your changes?

Reply:
the paragon changes (and potentially any needed package-lock changes thus generated by the new dep, will be needed for this PR

Reply:
but yeah package lock changes in the PR currently seem to be causing issues so they need an update

Reply:
typically running locallly `npm ci` and checking in that latest package lock will be best